src/main/java/io/codelex/arithmetic/HomeworkTasks/Exercise6.java

```diff
@@ -0,0 +1,40 @@
+package io.codelex.arithmetic.HomeworkTasks;
+
+//Write a program called CozaLozaWoza which prints the numbers 1 to 110,
+// 11 numbers per line. The program shall print "Coza" in place of the numbers which are multiples of 3,
+// "Loza" for multiples of 5, "Woza" for multiples of 7,
+// "CozaLoza" for multiples of 3 and 5, and so on.
+
+public class Exercise6 {
+    public static void main(String[] args) {
+        for (int i = 1; i <= 110; i++) {
+            boolean divideBy3 = (i % 3 == 0);
+            boolean divideBy5 = (i % 5 == 0);
+            boolean divideBy7 = (i % 7 == 0);
+
+
+            if (divideBy3 && divideBy5 && divideBy7) {
```

Comment:
Šito varēja nedaudz īsāk uzrakstīt (šis ir pseidokods, jebšu tīri lai ideju saprastu):
if (divideBy3) print Coza
if (divideBy5) print Loza
if (divideBy7) print Woza
if (!divideBy3 && !divideBy5 && !divideBy7) print i + " "